src/debian/control

```diff
@@ -3,7 +3,7 @@ Maintainer: John Billings <billings@yelp.com>
 Build-Depends:
     dh-virtualenv,
 
-Depends: python3.7, ${shlibs:Depends}
+Depends: python3.7, paasta-tools (>= 0.144.7), ${shlibs:Depends}
```

Comment:
TODO: bump this version to something that includes https://github.com/Yelp/paasta/pull/3450